proto/ibc/applications/transfer/v2/packet.proto

```diff
@@ -16,4 +16,6 @@ message FungibleTokenPacketData {
   string sender = 3;
   // the recipient address on the destination chain
   string receiver = 4;
+  // optional metadata
+  bytes metadata = 5;
```

Comment:
What's the reasoning for bytes instead of string? Just curious since all the other packet data fields are strings

Reply:
I figured someone may want to use protobufs for metadata, so they'd encode it as bytes